### DISCUSSION TITLE
EOL Conversion

### DISCUSSION THREAD

**Discussion by umar-afzaal** [dail8859/NotepadNext#602] [Ideas]
Im using notepadnext from my win host to open docs from a ssh session (unix). the default setting for the EOL Conversion param is always DOS.    There should be a way to permanently set it for all new documents. In my case, UNIX for all new docs since I mostly work from the ssh session?

**Comment by dail8859** [comment]
If I understand correctly...it sounds like there's 2 pieces here:

1. Auto detecting the current line endings of a file when it is opened
2. Specify the EOL mode to be used for new files

These aren't implemented yet but definitely make sense to have. If you want to open up a couple of issues that would be great so they can be tracked.
